microsite/src/main/resources/microsite/css/override.css

```diff
@@ -42,29 +43,33 @@
     background: url("../img/sidebar_brand.png") no-repeat;
 }
 
-@media (min--moz-device-pixel-ratio: 1.3), (-o-min-device-pixel-ratio: 2.6 / 2), (-webkit-min-device-pixel-ratio: 1.3), (min-device-pixel-ratio: 1.3), (min-resolution: 1.3dppx) {
+ @media (min--moz-device-pixel-ratio: 1.3), (-o-min-device-pixel-ratio: 2.6 / 2), (-webkit-min-device-pixel-ratio: 1.3), (min-device-pixel-ratio: 1.3), (min-resolution: 1.3dppx) {
     .sidebar-nav > .sidebar-brand a .brand-wrapper {
         /* on retina, use image that's scaled by 2 */
-        background-image: url("../img/sidebar_brand2x.png");
+    /*    background-image: url("../img/sidebar_brand2x.png"); */
         background-size: 111px 36px;
     }
 }
 
+
+
 #site-header .navbar-wrapper .brand .icon-wrapper {
     position: relative;
     width: 136px;
     height: 44px;
-    background: url("../img/navbar_brand.png") no-repeat;
+    background: url("../img/navbar_bran.png") no-repeat;
```

Comment:
typo?

Reply:
no.

Reply:
are you sure? because that looks to me like a typo and I don't think there's any file `navbar_bran.png`, I see only `navbar_brand.png`...

Reply:
Sorry for delay. I'm cheating a bit or else we will have two logos.